### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by maqnouch**

### CODE REVIEW

File: src/Symfony/Component/Serializer/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Serializer Component
 ====================
 
-With the Serializer component its possible to handle serializing data structures,
+With the Serializer component it’s possible to handle serializing data structures,
```
Comment (Tobion) [inline src/Symfony/Component/Serializer/README.md]: I don't think we use that kind of apostrophe anywhere.

Comment (pborreli) [inline src/Symfony/Component/Serializer/README.md]: indeed, replace with `'`

**Comment by dunglas** [comment]
Thanks for the fix!

Can you edit your PR description to add the Pull Request Template (https://github.com/symfony/symfony/blob/2.7/CONTRIBUTING.md) and add [Serializer] in your commit message?

**Comment by nicolas-grekas** [comment]
ping @maqnouch

**Comment by fabpot** [comment]
fixed it myself in 2.3.

**Pull request closed without merging**